examples/source/user-consent/Geolocation-based_Consent_Flow.html

```diff
@@ -75,51 +75,98 @@
 </head>
 <body>
   <!-- ## Setting up amp-geo -->
-  <!-- First we need to setup the [`amp-geo`](/content/amp-dev/documentation/components/reference/amp-geo-v0.1.md) extension. We'll use the present `eea` which includes all countries of the European Economic Area. -->
+  <!-- First we need to setup the [`amp-geo`](/content/amp-dev/documentation/components/reference/amp-geo-v0.1.md) extension. We'll use the iso code `us` which represents the United States, `preset-eea`, and [`unknown`](https://amp.dev/documentation/components/amp-geo/?format=websites#operation)-->
```

Comment:
nit: you can also link to https://amp.dev/documentation/components/amp-geo/?format=websites#preset-country-groups